### DISCUSSION TITLE
<title>Change photo on profile

### DISCUSSION THREAD

**Discussion by Carib909** [community/community#126305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

No place to change my photo which was added by within. It’s not me!

**Comment by talhaahussain** [comment]
Hey Carib909,

Are you referring to your profile picture? Your GitHub profile picture can be changed by going to your profile and clicking on your profile picture. This should take you to a page called "public profile " - from here, you should see an edit button on your profile picture. Click this button, and you'll have the option to upload a new photo. I hope this helps!

**Comment by gil-son** [comment]
Hello,

Try to follow these instructions in bellow

On the right side you can see your mini picture. Click on it
Now click on settings
Now click on public profile
And finally click on edit button to change your photo


If I supported you, consider to mark my answer. Good Luck!
